prow/config.yaml

```diff
@@ -18,6 +18,10 @@ triggers:
   - google/cadvisor
   trusted_org: kubernetes
 
+heart:
```

Comment:
Yeah, something like that makes sense. I'd like to redo much of prow's config, but I wanted to get a few examples of plugin configs first. Now is probably a good time to revisit that idea.